sdk/storage/Azure.Storage.Common/src/OperationInterruptedException.cs

```diff
@@ -0,0 +1,102 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Azure.Core;
+
+namespace Azure.Storage
+{
+#pragma warning disable CA2229, CA2235 // False positive
+    /// <summary>
+    /// An exception thrown when an operation is interrupted and can be continued later on.
+    /// </summary>
+    [Serializable]
+    public class OperationInterruptedException : RequestFailedException, ISerializable
+    {
+        /// <summary>
+        /// Gets the continuation token from the last successful response from the service.
+        /// </summary>
+        public string ContinuationToken { get; }
+
+        /// <summary>Initializes a new instance of the <see cref="RequestFailedException"></see> class with a specified error message.</summary>
+        /// <param name="message">The message that describes the error.</param>
+        public OperationInterruptedException(string message) : this(0, message)
```

Comment:
do we need all these constructors ?

Reply:
We don't. I think I was just copying `RequestFailedException`